components/MetadataEditor.vue

```diff
@@ -0,0 +1,283 @@
+                  mdi-plus-circle
+                </v-icon>
+              </v-subheader>
+              <v-list-item
+                v-for="(item, i) in metadata.People"
+                :key="i"
+                @click="handlePersonEdit(item)"
+              >
+                <v-list-item-avatar>
+                  <v-icon class="person-icon">mdi-account</v-icon>
+                </v-list-item-avatar>
+                <v-list-item-content>
+                  <v-list-item-title>
+                    {{ item.Name }}
+                  </v-list-item-title>
+                  <v-list-item-subtitle class="mt-1">
+                    {{ item.Role || item.Type }}
+                  </v-list-item-subtitle>
+                </v-list-item-content>
+                <v-list-item-action @click.stop="handlePersonDel(i)">
+                  <v-icon>mdi-delete</v-icon>
+                </v-list-item-action>
+              </v-list-item>
+            </v-list>
+          </v-tab-item>
+          <v-tab-item>
+            <image-editor :metadata="metadata"></image-editor>
+          </v-tab-item>
+        </v-tabs>
+      </v-form>
+    </v-card-text>
+
+    <v-card-actions class="d-flex justify-center align-center">
+      <v-btn @click="$emit('cancel')">Cancel</v-btn>
+      <v-btn color="primary" @click="saveMetadata">Save</v-btn>
+    </v-card-actions>
+    <person-editor
+      :person="person"
+      :dialog.sync="dialog"
+      @update:person="handlePersonUpdate"
+    ></person-editor>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { format, formatISO } from 'date-fns';
+import { pick } from 'lodash';
+import { BaseItemDto, BaseItemPerson } from '~/api';
+
+export default Vue.extend({
+  props: {
+    itemId: {
+      type: String,
+      default: ''
+    }
+  },
+
+  data() {
+    return {
+      metadata: {} as BaseItemDto,
+      saved: false,
+      menu: false,
+      dialog: false,
+      person: null as BaseItemPerson | null
+    };
+  },
+  computed: {
+    premiereDate: {
+      get() {
+        if (!this.metadata.PremiereDate) return '';
+        const dateStr = format(
+          new Date(this.metadata.PremiereDate),
+          'yyyy-MM-dd'
+        );
+        return dateStr;
+      }
+    },
+    dateCreated: {
+      get() {
+        if (!this.metadata.DateCreated) return '';
+        const dateStr = format(
+          new Date(this.metadata.DateCreated),
+          'yyyy-MM-dd'
+        );
+        return dateStr;
+      }
+    }
+  },
+  watch: {
+    itemId() {
+      this.fetchItemInfo();
+    }
+  },
+  created() {
+    this.fetchItemInfo();
+  },
+  methods: {
+    async fetchItemInfo() {
+      const userId = this.$auth.user.Id;
+      const itemInfo = (
+        await this.$userLibraryApi.getItem({
+          userId,
+          itemId: this.itemId
+        })
+      ).data;
+      this.$data.metadata = itemInfo;
+    },
+    async saveMetadata() {
+      const item = pick(this.metadata, [
```

Comment:
Because the input parameters and output parameters of this interface design are inconsistent, such as ForcedSortName and SortName. Not sure if there are other problems, so I copied the implementation of jf-web